features/cryptocell/FEATURE_CRYPTOCELL310/cmac_alt.h

```diff
@@ -0,0 +1,50 @@
+/*
+ *  aes_alt.h
+ *
+ *  Copyright (C) 2018, ARM Limited, All Rights Reserved
```

Comment:
Note that the dates and branding of this license header are out-of-date

Reply:
yes, copy\paste error as well